OpenOversight/app/main/views.py

```diff
@@ -433,8 +438,18 @@ def submit_complaint():
 @main.route('/submit', methods=['GET', 'POST'])
 @limiter.limit('5/minute')
 def submit_data():
-    departments = Department.query.all()
-    return render_template('submit_deptselect.html', departments=departments)
+    # try to use preferred department if available
+    try:
+        if User.query.filter_by(id=current_user.id).one().dept_pref:
+            department = User.query.filter_by(id=current_user.id).one().dept_pref
+            return redirect(url_for('main.submit_department_images', department_id=department))
```

Comment:
I think it's OK to do this but for logged-in users that might want to submit images for a department that is not their own, there should be a note on the submit page only displayed for users that have a default department that says "Want to submit for another department? Change your default department" where "default department" links to the change department form. What do you think?

Reply:
how about a `flash()` for this?

Reply:
✅